src/index.css

```diff
@@ -69,3 +70,68 @@ button:focus-visible {
     background-color: #F9F9F9;
   }
 }
+
+h2 {
+  font-size: 36px;
+  font-weight: 700;
+  color: #FFF;
+}
+
+@media screen and (min-width: 390px) and (max-width: 736px) {
+
+  h2 {
+    font-size: 19px;
+  }
+}
+
+p {
+  font-size: 22px;
+  color: #FFF;
+}
+
+@media screen and (min-width: 390px) and (max-width: 736px) {
```

Comment:
If you remove `min-width`, you can fix this 
<img width="848" alt="image" src="https://github.com/3apy6a/fe-react-2024/assets/98031768/3376bc65-e4d3-4545-90dd-0da226663ab3">

Reply:
done